server/sonar-server/src/main/java/org/sonar/server/ui/ws/ComponentNavigationAction.java

```diff
@@ -235,7 +235,8 @@ private void writeConfigPageAccess(JsonWriter json, boolean isAdmin, ComponentDt
     json.prop("showQualityProfiles", isProject);
     json.prop("showQualityGates", isProject);
     json.prop("showManualMeasures", isAdmin);
-    json.prop("showActionPlans", isAdmin && isProject);
+    // TODO delete showActionPlans when UI is updated
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)